hathor/p2p/states/hello.py

```diff
@@ -102,4 +102,17 @@ def handle_hello(self, payload: str) -> None:
             return
 
         protocol.app_version = data['app']
+
+        from hathor.p2p.netfilter import get_table
+        from hathor.p2p.netfilter.context import NetfilterContext
+        context = NetfilterContext(
+            protocol=self.protocol,
+            connections=self.protocol.connections,
+            addr=self.protocol.transport.getPeer(),
+        )
+        verdict = get_table('filter').get_chain('post_hello').process(context)
+        if not bool(verdict):
+            self.protocol.close_connection()
```

Comment:
Should we have a `reason` on `close_connection` so we can better debug why the connection was closed?

Reply:
Done! Can you review it again, please?

Reply:
Looks great!